docs/product-user-guide/src/main/asciidoc/business-central-settings-setting-priorities-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='business-central-settings-setting-priorities-proc']
+= Setting priorities
+
+A user can have multiple roles and belong to multiple groups. The Priority setting determines the order of precedence of a role or group.
+
+.Procedure
+Select a priority  from the Priority menu, then click *Save*. If a user has a role or belongs to a group that has conflicting settings, the settings of the role or group with the highest priority applies.
```

Comment:
Not a huge deal (I don't care), but IBM says to say "and then" in a series, not just "then". So "menu, and then". If you're starting a new sentence, though, like in my example a few comments up, you can start "Then". Your call.